python-avd/pyavd/api/_anta/avd_fabric_data.py

```diff
@@ -60,7 +60,11 @@ def from_structured_config(cls, structured_config: dict[str, Any]) -> AvdDeviceD
         }
 
         # Get the Loopback0 IP
-        loopback0_ip = get(get_item(get(structured_config, "loopback_interfaces", []), "name", "Loopback0", default={}), "ip_address")
+        loopback0_ip_str = get(get_item(get(structured_config, "loopback_interfaces", []), "name", "Loopback0", default={}), "ip_address")
+        loopback0_ip = ip_interface(loopback0_ip_str).ip if loopback0_ip_str else None
```

Comment:
Something like this should remove the need for line 65

```suggestion
        loopback0_ip = IPv4Address(loopback0_ip_str) if loopback0_ip_str else None
```

Reply:
Fixed